### DISCUSSION TITLE
Song Location of Osu Lazer not changing despite changing lazer install location

### DISCUSSION THREAD

**Discussion by BlazeBG12** [ppy/osu#26898] [Q&A]
When I am on the First time setup screen, every time I try to move the install directory which should move where everything is stored, such as stored beatmaps, skins, and score replay files. This however, does not work. When I installed lazer earlier today, before importing my maps from stable, I made sure to change the install directory which restarted my game and I was once again back in the setup wizard. 

I tried to import my maps, however they were still being imported to my C drive instead of the D drive where I had moved the install folder. I read over the page for importing maps again and it said that if the Stable file directory is on the same drive as Lazer, there shouldn't be any file duplication. My stable install is on the D drive where I placed the new Lazer install. I even tried deleting the C drive lazer install manually, only to see that when I clicked import, the Lazer folder was returned back to my C drive, while I also had the one on the D drive and song files and everything just goes to C instead of D drive. 

Even if a Second copy of the files does exist, I am fine with that, however I want it to be in the correct directory which it sadly isn't when I attempt to do that. 

TLDR: Step by step of what I did: 
1. Install Lazer
2. Select correct directory where my Osu install is ( D drive)
3. Change lazer directory to be on the same drive C -> D
4. restart game
6. Go to C and delete residual files from old lazer install as it's now supposed to be in the D drive
7. Click import
8. Everything is being imported back to C drive and another lazer install folder exists even after it's been deleted

OS - Windows 10
